pr_testing/test_multiple_prs.sh

```diff
@@ -986,6 +986,9 @@ if [ "X$DO_SHORT_MATRIX" = Xtrue -a "X$BUILD_OK" = Xtrue -a "$ONLY_FIREWORKS" =
       GPU-*)
         WF_LIST="$MATRIX_EXTRAS"
         ;;
+      PROFILING-*)
+        WF_LIST="$MATRIX_EXTRAS"
+        ;;
```

Comment:
CONTEXT will not be PROFILING-* . so this change is not needed. 
You should check $ENABLE_BOT_TESTS and if `profiling` is available in it then run an extra test. e.g. you need a section like https://github.com/cms-sw/cms-bot/blob/master/pr_testing/test_multiple_prs.sh#L1149-L1160 where you check if profiling is requested then start a profiling job.